### PR TITLE
Flesh out Contributing.md

### PR DESCRIPTION
**Pull request by bfolie**
# Citrine Python PR

## Description 
Flesh out instructions for contributing to the repo (Issue #79)

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
-For additional (non-binding) inspiration, check out the [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
\ No newline at end of file
+Type hints are strongly encouraged.
+
+Docstrings must follow [Numpy style](https://numpydoc.readthedocs.io/en/latest/format.html) so that Sphinx can parse them to make the docs.
+
+For additional (non-binding) inspiration, check out the [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
+
+## PR Submission
+Features should be added in a branch with a descriptive name and the pull requests (PR) submitted into the `develop` branch.
```
Comment (bfolie) [inline CONTRIBUTING.md]:
```suggestion
Features should be developed in a branch with a descriptive name and the pull request (PR) submitted into the `develop` branch.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
 
 ## Coding Style
 The citrine-python library follows [PEP8](https://www.python.org/dev/peps/pep-0008/), with the following exceptions:
-* Maximum line length is 99 characters
+* Maximum line length is 99 characters for code and 119 characters for docstrings
```
Comment (latture) [inline CONTRIBUTING.md]: Is this also intended for files in the `docs` folder? I find it's easiest to organize my thoughts when I use 1 sentence/line. It also makes it easier to edit later. However, this sometimes results in lines that exceed 119 characters.

Comment (bfolie) [reply CONTRIBUTING.md]: The docs/ folder is not checked by the linter, so this is not a strict requirement. But I do find it helpful to break up long sentences into multiple lines (to me, the important part is not having more than 1 sentence per line)

Comment (latture) [reply CONTRIBUTING.md]: 👍 
Thanks for the clarification. That was not clear to me.

**Pull request merged**